src/main/java/frc/robot/commands/DefaultMechCommand.java

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.LauncherConstants;
+import frc.robot.constants.AimingConstants.AimState;
+import frc.robot.constants.LauncherConstants.LauncherMode;
+import frc.robot.subsystems.AimingSubsystem;
+import frc.robot.subsystems.Input;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.LauncherSubsystem;
+import frc.states.MechState;
+import frc.states.MechStates.Intook;
+import frc.states.MechStates.ReadyForAim;
+import frc.states.MechStates.ReadyForHandoff;
+import frc.states.MechStates.ReadyForIntake;
+import frc.states.MechStates.ReadyForLaunch;
+
+public class DefaultMechCommand extends Command {
+    private final IntakeSubsystem mIntakeSubsystem;
+    private final LauncherSubsystem mLauncherSubsystem;
+    private final AimingSubsystem mAimingSubsystem;
+
+    private LauncherMode mLauncherMode = LauncherMode.OFF;
+    private AimState mAimState = AimState.STOW;
+
+    private final MechState readyForIntake;
```

Comment:
Can you rename to follow the naming convention for fields please? So mReadyForIntake for example.